assets/base.css

```diff
@@ -1057,6 +1057,27 @@ summary::-webkit-details-marker {
   }
 }
 
+@media screen and (max-width: 989px) {
+  .grid--1-col-tablet-down .grid__item {
+    width: 100%;
+    max-width: 100%;
+  }
+
+  .slider--tablet.grid--peek.grid--1-col-tablet-down .grid__item {
+    width: 100%;
```

Comment:
Same thing for the featured collection.

Reply:
Since on the declaration above we're already applying a `width: 100%;` I don't think it's needed there anyway. So maybe it can just be removed ? 🤔

Reply:
I just removed it and it doesn't seem like we need it since width is accounted in other media queries 👍 It's something we are adding and not something that was on main as well